kotest-assertions/src/jvmTest/kotlin/com/sksamuel/kotest/matchers/date/instantMatcherTest.kt

```diff
@@ -6,21 +6,6 @@ import java.time.Instant
 
 class InstantMatcherTest : FreeSpec() {
    init {
-      "same instance of instant should be same" {
-         val currentInstant = Instant.now()
-         currentInstant shouldBe currentInstant
-      }
-
-      "different instance of instant having same time should be same" {
-         Instant.ofEpochMilli(10000) shouldBe Instant.ofEpochMilli(10000)
-      }
-
-      "instance of different time should be different" {
-         val currentInstant = Instant.now()
-         val pastInstant = currentInstant.minusMillis(40000)
-         currentInstant shouldNotBe pastInstant
-      }
-
```

Comment:
I suppose we could keep these tests, just to make sure

Reply:
Done